diff --git a/src/RingDecomposerLib/CMakeLists.txt b/src/RingDecomposerLib/CMakeLists.txt
index f032c12..8f8f8f3 100644
--- a/src/RingDecomposerLib/CMakeLists.txt
+++ b/src/RingDecomposerLib/CMakeLists.txt
@@ -15,6 +15,7 @@ function(build_library LIBRARY_NAME SPECIFIER)
               RDLbitset.c)
   set_property(TARGET "${LIBRARY_NAME}" PROPERTY C_STANDARD 90)
   set_target_properties("${LIBRARY_NAME}" PROPERTIES POSITION_INDEPENDENT_CODE True)
+  set_target_properties("${LIBRARY_NAME}" PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS True)
   if(NOT WIN32)
     target_link_libraries("${LIBRARY_NAME}" m)
   endif()
